Example/Sources/AuthViewController.swift

```diff
@@ -74,7 +75,10 @@ class AuthViewController: UIViewController {
         ])
 
         // Method 2: Use WKWebView
-        webView.load(URLRequest(url: URL(string: "https://demo.virtusize.com")!))
+		let url = VirtusizeBranch.applyBranch(
+			to: URL(string: "https://demo.virtusize.com")!,
+			branch: "test-branch")
```

Comment:
@akueisara I would appreciate your advice here.

Even though the `branch` concept is a shared idea between all modules and naturally should belong to the `core` module, the URL-modified approach is only specific to WebView based apps which is specific to `auth` module. (_Native apps will use different approach, and be delivered in another PR_)

Also, developers would need to `import VirtusizeCore` which I don't really like, as there was no real need to explicitly use `core` module for WebView based apps before.

Do you think the current implantation is fine? Or it's worth to introduce a method to `VirtusizeAuth`?
For example:
```swift
// VirtusizeAuth project > URL+Extensions.swift file
extension URL {
    public func applyBranch(_ branchName: String) -> URL {
    }
}
```

Reply:
Wouldn't importing `Virtusize` solve the issue?
Virtusize embeds both VirtusizeCore and VirtusizeAuth.
We just need to ensure that the CocoaPods spec is configured correctly.

Reply:
I don't remember it well, but there's a file called VirtusizeExports.swift which exposes VirtusizeAuth and VirtusizeCore

Reply:
@akueisara I have two concerns about it:
1. Importing `Virtusize` into `VirtusizeAuth` will introduce cyclic dependency, which might work, but need to check.
2. Having `VirtusizeAuth` independent from `Virtusize` keep the import size lower for the WebView-native apps. Seems they only need to import `VirtusizeAuth` to make the authentication work, and there is no need for `Virtusize` package.

Let me take another look if we could achieve what we need without `Virtusize` package. Seems Android implementation allows to achieve quite easy.

Reply:
I see. It makes sense. I didn't paying attention that this was only for WebView based apps.

Reply:
I propose to go with the current PR implementation. We can improve it later if there is a demand.